src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -36,9 +34,9 @@ public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> att
         {
             TagName = tagName;
             Attributes = new Dictionary<string, string>(attributes, StringComparer.OrdinalIgnoreCase);
-            PreContent = string.Empty;
-            _content = string.Empty;
-            PostContent = string.Empty;
+            PreContent = new DefaultTagHelperContent();
```

Comment:
This is exactly the kind of thing to only do if profiling shows a benefit. Please leave it for now.